src/phpDebug.ts

```diff
@@ -197,8 +197,9 @@ class PhpDebugSession extends vscode.DebugSession {
 
     /** converts a server-side XDebug file URI to a local path for VS Code with respect to source root settings */
     protected convertDebuggerPathToClient(fileUri: string): string {
-        // convert the file URI to a path
-        const serverPath = decodeURI(url.parse(fileUri).pathname.substr(1));
+        // convert the file URI to a path. Don't remove starting slash on unix platforms.
+        let n:number = (process.platform === 'win32') ? 1 : 0;
```

Comment:
Hey Felix,

This maps the server path returned to us to the client path. This modified path doesn't get sent back to the server so it makes sense at this point to change the path.

The reason for the different `substr` index is to keep the starting slash on Unix systems so we can map `file:///var/www` to `/var/www` (otherwise we'd have an invalid var/www). Windows won't want that starting slash so I imagine that's why you took it out initially. 

I can tell this works for linux to linux though I'm not sure how it would work for Windows to Unix or Windows to Windows.